Makefile

```diff
@@ -1,29 +1,45 @@
 JSONNET_FMT := jsonnetfmt -n 2 --max-blank-lines 2 --string-style s --comment-style s
 SHELL := /bin/bash
 
+install-ci-deps:
+	go install github.com/google/go-jsonnet/cmd/jsonnet@v0.18.0
+	go install github.com/google/go-jsonnet/cmd/jsonnetfmt@v0.18.0
+	go install github.com/google/go-jsonnet/cmd/jsonnet-lint@v0.18.0
+	go install github.com/monitoring-mixins/mixtool/cmd/mixtool@ae18e31161ea
+	go install github.com/jsonnet-bundler/jsonnet-bundler/cmd/jb@v0.4.0
+
 fmt:
-		@find . -name '*.libsonnet' -print -o -name '*.jsonnet' -print | \
-				xargs -n 1 -- $(JSONNET_FMT) -i
+	@find . -name '*.libsonnet' -print -o -name '*.jsonnet' -print | \
+			xargs -n 1 -- $(JSONNET_FMT) -i
+
+lint-fmt:
```

Comment:
nit: maybe these targets could depend on `install-ci-deps` to help people running locally?

Reply:
It's a bit rude to run `go install` on people's machines, isn't it? 😆  I'll leave it as-is for now

Reply:
fair enough